test/tests/ver-1d/ver-1d-trapping.i

```diff
@@ -119,6 +123,12 @@ temperature = 1000
     boundary = right
   []
 []
+[Functions]
+  [BC_func]
+    type = ParsedFunction
+    expression = '${fparse 3.1622e18 / cl}*tanh( 3 * t )'
```

Comment:
```suggestion
    expression = '${fparse cl / cl}*tanh( 3 * t )'
```
Or create a new parameter at the beginning of the input file to define the value at the BC, and have it equal to cl.

Reply:
Fixed